dlrs/main/lwc/rollupEditor/rollupEditor.html

```diff
@@ -1,269 +1,542 @@
-            <c-rollup-editor-error
-              errors={errors.AggregateOperation__c}
-            ></c-rollup-editor-error>
-          </lightning-layout-item>
-          <lightning-layout-item size="4" class="slds-form-element_stacked">
-            <lightning-input
-              name="rollup_AggregateResultField__c"
-              data-name="rollup_AggregateResultField__c"
-              label="Aggregate Result Field"
-              placeholder="Total__c"
-              value={rollup.AggregateResultField__c}
-            ></lightning-input>
-            <c-rollup-editor-error
-              errors={errors.AggregateResultField__c}
-            ></c-rollup-editor-error>
-          </lightning-layout-item>
-        </lightning-layout>
-        <lightning-layout class="slds-p-horizontal_medium">
-          <lightning-layout-item size="4" class="slds-form-element_stacked">
-            <h2 class="header">Aggregate All Rows</h2>
-            <lightning-input
-              type="checkbox"
-              name="rollup_AggregateAllRows__c"
-              data-name="rollup_AggregateAllRows__c"
-              label="Include Deleted"
-              value={rollup.AggregateAllRows__c}
-            ></lightning-input>
-            <c-rollup-editor-error
-              errors={errors.AggregateAllRows__c}
-            ></c-rollup-editor-error>
-          </lightning-layout-item>
-          <lightning-layout-item size="4" class="slds-form-element_stacked">
-            <lightning-input
-              type="number"
-              name="rollup_RowLimit__c"
-              data-name="rollup_RowLimit__c"
-              label="Row Limit"
-              placeholder="0"
-              value={rollup.RowLimit__c}
-            ></lightning-input>
-            <c-rollup-editor-error
-              errors={errors.RowLimit__c}
-            ></c-rollup-editor-error>
-          </lightning-layout-item>
-          <lightning-layout-item size="4" class="slds-form-element_stacked">
-            <lightning-input
-              type="text"
-              name="rollup_ConcatenateDelimiter__c"
-              data-name="rollup_ConcatenateDelimiter__c"
-              label="Concatenate Delimiter"
-              value={rollup.ConcatenateDelimiter__c}
-            ></lightning-input>
-            <c-rollup-editor-error
-              errors={errors.ConcatenateDelimiter__c}
-            ></c-rollup-editor-error>
-          </lightning-layout-item>
-        </lightning-layout>
-        <lightning-layout horizontal-align="center">
-          <lightning-layout-item>
-            <lightning-button
-              label="Save"
-              variant="brand"
-              onclick={runSave}
-            ></lightning-button>
-          </lightning-layout-item>
-        </lightning-layout>
-      </div>
-      <div class="slds-box">
-        <c-rollup-status-check name={rollup.Id}></c-rollup-status-check>
+          </div>
+        </div>
+
+        <!-- Rollup Details Section -->
```

Comment:
How do you feel about starting the wizard with this open? I have a concern about putting required fields in a collapsed-by-default wizard.